src/components/TermsAndConditions.tsx

```diff
@@ -1,5 +1,65 @@
 function TermsAndConditions() {
-  return <h1>Terms & Conditions</h1>;
+  const termsData = [
```

Comment:
At the moment this `termsData` array is recreated on every rerender of the component. However, it doesn't change, the value is constant. So, I suggest you move it outside of the component because then it's only created once and just referenced on every rerender.

This doesn't have any significant memory or performance impact, but it's good to be aware of which values are rerender dependent (because they reference props or state) and which aren't – and those that aren't don't need to be inside the component code.